src/main/java/org/carlmontrobotics/Constants.java

```diff
@@ -447,10 +446,17 @@ public static final class Manipulator {
 			public static final int EJECT_RPM = Button.kX.value;
 			public static final int RAISE_CLIMBER = Button.kA.value;
 			public static final int LOWER_CLIMBER = Button.kY.value;
-			
+			public static final int PASS_TO_OUTTAKE_STICK =
+					Button.kLeftStick.value;
+			public static final int PASS_TO_INTAKE_STICK =
+					Button.kRightStick.value;
 		}
 
 		public static final double JOY_THRESH = 0.01;
 		public static final double MIN_AXIS_TRIGGER_VALUE = 0.2;// woah, this is high.
+		public static final int UP_D_PAD = 0;
+		public static final int DOWN_D_PAD = 180;
+		public static final int LEFT_D_PAD = 270;
+		public static final int A_BUTTON = Button.kA.value;
```

Comment:
Typically, we name such constants with what command they are associated with. For example,
`public static final int RAISE_CLIMBER = 0; // Up D-pad`
btw your `RAISE_CLIMBER` variable on line 447 isn't actually used anywhere, so please double check that the button mappings are all updated.

Reply:
I find @aaron345678's approach much more useful. It means that if I want to know which button runs a particular command I can just look for where it is referenced and I immediately know. With @ProfessorAtomicManiac's approach I additionally need to find the button constant in Constants to see what the comment says. I don't see what benefit that provides.

Reply:
My thinking was that I can go to `Constants.java`, and see all the buttons. I can see which button (what the variable is assigned) runs a particular command (which is in the variable name). This is what we've done in the past.
Correct me if I'm wrong but using Aaron's way would mean I can go to `Constants.java`, find that button then can use VSCode features to find where that button is referenced in the code to know what the button runs. Or I can go to `RobotContainer.java`, and Ctrl+F to find where that button is used.

Tbh, I don't care which way we use and that should really be left up to @FriedLongJohns. I've been really nitpicky on conventions and uniformity which isn't exactly necessary. The important thing that I want @aaron345678 to do is to get rid of unused constants like `RAISE_CLIMBER` so people don't try to change that variable thinking it changes the button to raise the climber.